draft-ietf-rtcweb-jsep.xml

```diff
@@ -3610,18 +3610,26 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         applied, processing MUST stop and an error MUST be returned.
         <list style="symbols">
 
-          <t>If the ICE ufrag or password changed from the previous
-          remote description, then an ICE restart is needed, as
-          described in Section 9.1.1.1 of
-          <xref target="RFC5245" /> If the description is of type
-          "offer", mark that an ICE restart is needed. If the
-          description is of type "answer" and the current local
-          description is also an ICE restart, then signal the ICE agent
-          to begin checks as described in Section 9.3.1.1 of
-          <xref target="RFC5245" />. An answer MUST change the ufrag
-          and password in an answer if and only if ICE is restarting,
-          as described in Section 9.2.1.1 of
-          <xref target="RFC5245" />.</t>
+          <t>If the PeerConnection state is "have-local-offer", and the
+          ICE ufrag or password changed from the previous remote
+          description, then an ICE restart is needed, as described in
+          Section 9.1.1.1 of <xref target="RFC5245" />. If the
+          description is of type "offer", mark that an ICE restart is
+          needed. If the description is of type "answer" or "pranswer"
+          and the current local description is also an ICE restart, then
+          signal the ICE agent to begin checks as described in Section
+          9.3.1.1 of <xref target="RFC5245" />. An answer MUST change
```

Comment:
It said "answer" twice before this PR. :)

I'll fix it, but see above comment about moving it.